iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -723,7 +799,8 @@ export class Viewer {
     }
 
     _setTitle(docIndex) {
-        $('#top-bar .document-title').text($('head title', this._iframes.eq(docIndex).contents()).text());
+        const mode = this._iv.options.reviewMode ? " - Review Mode" : "";
```

Comment:
We may want to add this in the future (and even a button to switch between modes), but for now we should leave the title unchanged